src/main/resources/jenkins/security/plugins/ldap/validation/validate.jelly

```diff
@@ -78,6 +78,7 @@
       <img src="${imagesURL}/spinner.gif"/>
       ${attrs.progress}
     </div>
+    <!-- TODO: remove once pom is based off 2.361-->
```

Comment:
IMO it is better to just change the `jenkins.version` to 2.361.x and delete the tech debt.

If you want to offer a fix to 2.346.x users, do it on a branch.

Reply:
I don't have rights to create a branch. @olamy @jtnord WDYT ?

Reply:
@Dohbedoh  if you don't have write access no you cannot. I can do it for you. (It will cost you few croissants to deliver to my place 😜)
what branch name do you want?

Reply:
done https://github.com/jenkinsci/ldap-plugin/tree/2.12.x

Reply:
Thanks!